src/api-binder.ts

```diff
@@ -875,7 +875,7 @@ export class APIBinder {
 				delay: retryDelay,
 			});
 			await Bluebird.delay(retryDelay);
-			this.provisionOrRetry(retryDelay);
+			return this.provisionOrRetry(retryDelay);
```

Comment:
I think this should probably be `await` as we don't actually want to return a value and just want to wait on the promise chain (but either version will work for our purposes)